### ISSUE TITLE
Add back axiomatization for mitotic/meiotic specific phases

### ISSUE THREAD

**Issue by cmungall** [geneontology/go-ontology#12120]
'mitotic G1 phase' lacks equivalence axiom, causing https://build.berkeleybop.org/view/GO/job/build-go-assert-inferences-report/lastSuccessfulBuild/artifact/assert-report.txt/*view*/ to report:

```
EXISTS, TAGGED-INFERRED, NOT-ENTAILED	GO:0000080 'mitotic G1 phase' GO:0098763 'mitotic cell cycle phase'	 ! Direct SuperClasses for GO:0000080: GO:0051318 'G1 phase'
```

There are many similar ones

**Comment by dosumis** [comment]
See #11992.  Most edits done. Need to check status and how relates to this issue.

**Comment by dosumis** [comment]
Issue is (inferred) parent class using happens_during & (inferred) subclass po.  Fixing.

**Comment by dosumis** [comment]
Done.  For complete inference need to add more reasoning over timing relations to RO.  

oborel/obo-relations/issues#86

**Comment by cmungall** [comment]
OK, so when this is done we should lose these:

```
EXISTS, TAGGED-INFERRED, NOT-ENTAILED	GO:0007132 'meiotic metaphase I' GO:0098764 'meiosis I cell cycle phase'	 ! Direct SuperClasses for GO:0007132: GO:0051323 'metaphase'
EXISTS, TAGGED-INFERRED, NOT-ENTAILED	GO:0007133 'meiotic anaphase I' GO:0098764 'meiosis I cell cycle phase'	 ! Direct SuperClasses for GO:0007133: GO:0051322 'anaphase'
EXISTS, TAGGED-INFERRED, NOT-ENTAILED	GO:0007134 'meiotic telophase I' GO:0098764 'meiosis I cell cycle phase'	 ! Direct SuperClasses for GO:0007134: GO:0051326 'telophase'
```

**Comment by cmungall** [comment]
I made a new release of RO. Reopening this ticket and assigning to me. If the ETINEs above disappear I will close this.

**Comment by cmungall** [comment]
Assigning back to you David - even with the property chain in the tracker I can't recapitulate the inference

(if when you get it to work can you put a screenshot of the explanation here that'd be awesome)

**Comment by dosumis** [comment]
Following discussion in email: I'll switch back to PO between phases so that relationships between phases are displayed in QuickGO/Amiago.  Meiotic phase I and II classification will need to be asserted, but this is no big deal.  I'll manually remove is_inferred tags.

**Comment by dosumis** [comment]
Fixed see #12242
